src/jit/regset.cpp

```diff
@@ -45,6 +45,20 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 */
 
+void RegSet::verifyRegUsed(regNumber reg)
```

Comment:
These two methods need function headers. See https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#9.4.1.

Under the "Notes" section I would add a note saying that this method is intended to be called during code generation, and should simply validate that the register (or registers) have already been added to the modified set. Then, add a `TODO-Cleanup:` comment saying that we need to identify the places where this has not been done.